tests/infer/test_gradient.py

```diff
@@ -70,10 +70,10 @@ def guide(subsample):
     inference = SVI(model, guide, optim, loss=elbo)
     with xfail_if_not_implemented():
         if subsample_size == 1:
-            inference.loss_and_grads(model, guide, subsample=torch.LongTensor([0]))
```

Comment:
is there a difference here? wont it default to the right device?

Reply:
Its really odd, but it doesn't:

```python
In [1]: import torch

In [2]: torch.set_default_tensor_type('torch.cuda.DoubleTensor')

In [3]: torch.LongTensor(1)
Out[3]: tensor([140263273888632], device='cpu')

```